orttraining/orttraining/core/graph/gradient_builder.cc

```diff
@@ -96,6 +96,11 @@ IMPLEMENT_GRADIENT_BUILDER(GetTanhGradient) {
               {GI(0)})};
 }
 
+IMPLEMENT_GRADIENT_BUILDER(GetTriluGradient) {
```

Comment:
As we have added gradient builder, we should also add tests in gradient_ops_test.cc